sdk/tests/js/tests/tex-image-and-sub-image-2d-with-webgl-canvas.js

```diff
@@ -213,18 +228,24 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
     function runTest()
     {
-        var ctx = wtu.create3DContext();
-        var canvas = ctx.canvas;
+        // Note: We use preserveDrawingBuffer:true to prevent canvas
+        // visibility from interfering with the tests.
+        var ctx = wtu.create3DContext(null, { preserveDrawingBuffer:true });
```

Comment:
Can we only set preserveDrawingBuffer:true for the test cases where visible: true? This can be easily done by creating another canvas.

Otherwise we lose test coverage.

Reply:
Done.